src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c

```diff
@@ -856,16 +855,15 @@ void EnZoraegg_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s*
     }
 }
 
-TexturePtr D_80B33950[] = { object_zoraegg_Tex_003430, object_zoraegg_Tex_004430, object_zoraegg_Tex_004830 };
-
 void func_80B33818(Actor* thisx, PlayState* play) {
+    static TexturePtr sZoraBabyEyeTextures[] = { gZoraBabyEyeOpenTex, gZoraBabyEyeHalfTex, gZoraBabyEyeClosedTex };
```

Comment:
I personally like large arrays like this outside the function if possible, but it just comes down to styling choice, so it's not a big deal either way